src/proof-of-delivery-view/proof-of-delivery-view.controller.js

```diff
@@ -177,20 +176,18 @@
          * Prints the proof of delivery.
          */
         function printProofOfDelivery() {
+            var printer = new ProofOfDeliveryPrinter();
+
+            printer.openTab();
 
             (vm.proofOfDelivery.isInitiated() ? vm.proofOfDelivery.save() : $q.resolve(vm.proofOfDelivery))
                 .then(function(proofOfDelivery) {
-                    var popup = $window.open('', '_blank');
```

Comment:
we don't need a new window for report?

Reply:
No it is not need it is handled by printer.openTab(); in the line above(181)